projects/cairo/project.yaml

```diff
@@ -0,0 +1,6 @@
+homepage: https://gitlab.freedesktop.org/cairo/cairo
+language: c
+primary_contact: security-tps@google.com
```

Comment:
Try getting actual developer email. I dont think this mailing list is correct to add him, check with Julien, Jose. @Google-Autofuzz

Reply:
I contacted the upstream yesterday based on your comment: https://gitlab.freedesktop.org/cairo/cairo/-/merge_requests/69#note_705957

I'll update this as soon as I hear back from them.